substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1524,13 +1534,10 @@ mod remote_tests {
 
 	#[tokio::test]
 	async fn can_fetch_all() {
-		if std::option_env!("TEST_WS").is_none() {
-			return
-		}
```

Comment:
Did you intentionally enabled them? Before they were disabled if `TEST_WS` wasn't set?

Reply:
Good point. 

It makes sense to **only** use `TEST_WS` for larger tests vs defaulting to `DEFAULT_HTTP_ENDPOINT` which is `https://rpc.polkadot.io:443`

Added these conditionals back in.